uint16 out1
uint16 out2
---
